core/src/main/scala/filodb.core/memstore/PartKeyIndex.scala

```diff
@@ -132,7 +343,7 @@ trait PartKeyIndexRaw {
   /**
    * Commit index contents to disk
    */
-  def commit(): Long
+  def commit(): Unit
```

Comment:
Return type changed to avoid exposing Lucene implementation.  No one was using it.